tasks/create_static_services.yml

```diff
@@ -0,0 +1,10 @@
+---
+# Create static routes for services of an app
+
+- name: "Make sure static service exists for the '{{ app.name }}' app with all prefixes"
+  include_tasks: tasks/deploy_patch_static_service.yml
+  loop: "{{ blue_green_route_prefixes }}"
```

Comment:
we should maybe rename this variable in `blue_green_prefixes` ?

Reply:
:+1:

Reply:
:thinking: not sure as the prefixes are targeting routes?

Reply:
Now these prefixes are used for routes _and_ static services (e.g. route `app-next` points to service `app-next`)